secant/Features/Settings/SettingsView.swift

```diff
@@ -41,6 +53,77 @@ struct SettingsView: View {
     }
 }
 
+extension SettingsView {
+    private func exportAndShareLogs() {
+        exportLogsDisabled = true
+        
+        Task { @MainActor [self] in
```

Comment:
On a second read, I noticed this will execute on the main thread. I think is a good candidate to actually make it async and benefit from some parallelism. You can `async let` each log file and collect the results as they are done.

Reply:
This method is no longer available in this form, has been changed to async function and the TCA is taking care of it.